docs/user/kubevirt/install_ipi.md

```diff
@@ -0,0 +1,162 @@
+
+
+## Minimum resources
+
+The default master/worker:
+- 4 CPUs
+- 16 RAM
+- 120 GB disk
+
+For 3 masters/3 workers, the target Cluster **must have at least**:
+- 96RAM
+- 24vCPUs
+- 720GiB storage
+
+> Worker count can be reduced to 2 in `install-config.yaml` in case needed.
+
+The installer will create by default 1 bootstrap, 3 masters, and 3 workers machines. 
+By the time the first worker is up the bootstrap VM should be destroyed, and this is included in the minimum resources calculation.
+
+
+## Install 
+
+### Minimum permission for installation
+
+In order to be able to complete full installation process, the user running the installer must have privilaged for minimum permission.<br/>
+Please refer the the [Role yaml](./role.yml) to view the list of the required permissions.<br/>
+This role can be added to the cluster, and then binded to the user which running the installer, it can be done using the following ways:
+- Create RoleBinding which bind the role to the user in the specific namespace
+- If using Openshift as InfraCluster, run:<br/>
+`oc adm policy add-role-to-user <role> <user> -n <namespace>`
+
+
+### kubevirt-credentials
+
+Before you start the tenant cluster installation, make sure to login to the infra cluster, using one of these ways:
+
+1. For configuration, kubectl looks for a file named config in the $HOME/.kube directory. You can specify other kubeconfig files by setting the KUBECONFIG environment variable or by setting the --kubeconfig flag.
+2. Using the OpenShift oc command-line client (`oc login` command)
+https://docs.openshift.com/container-platform/4.6/cli_reference/openshift_cli/getting-started-cli.html#cli-logging-in_cli-developer-commands
+
+
+### Install using the wizard 
+
+At this stage, as the Kubevirt provider defined a dev preview, the installer can't be started using the wizard, please refer to [Install using Install Config yaml](#instal-using-install-config-yaml)
+
+
+### Install using Install Config yaml
+
+- Copy the example [install-config.yaml](./install-config.yaml#L13) file to `install_dir`
+- Change the following values
+>- [baseDomain](./install-config.yaml#L2) - Enter the base domain would be used for this cluster
+>- Add the machines resources size, in case you want other values than defaults (optional fields)<br/>
+>>> Should be following to this structure (remove `{}`):<br/>
+>>> ```console
+>>>     kubevirt:
+>>>       cpu: 4
+>>>       memory: 16G
+>>>       storageSize: 120Gi
+>>> ``` 
+>>> Set different values for compute machines (workers) and for controlplan machines (masters):
+>>>- [workers](./install-config.yaml#L7)
+>>>- [masters](./install-config.yaml#L13)
+>- Change the [workerReplicas](./install-config.yaml#L8) - In case you need copute machines replica different than 3
+>- Change the [masterReplicas](./install-config.yaml#L14) - In case you need controlplan machines replica different than 3
```

Comment:
Is it really true - can one install a cluster with 2 or 4 master ?

Reply:
Theoretically yes, never tried that